README.md

```diff
@@ -54,17 +54,13 @@ enc.signin(function (data) {
   console.log('OK with data')
   console.log(data)
 
-  // Get the categories
-  // enc.getCategories()
+  enc.getCategories()
 
-  // Get the keys for a category
-  // enc.getKeys('Category')
+  enc.getKeys('Category')
 
-  // Get the key details
-  // enc.getKeyDetails(<key id>)
+  enc.getKeyDetails(<key id>)
 
-  // Get the key password
-  // enc.getPassword(<key id>)
+  enc.getPassword(<key id>)
```

Comment:
Since all methods return something I suggest we comment them and leave the comments. What do you think?

Reply:
I would think that given this piece, is an example anyone that wishes to try it should be able to copy paste and test, without amending. 

The comments maybe an overkill especially since the code is more than self explanatory. If the intention is to showcase an example that cannot be used, consider using pseudo code or just comments (?)

Reply:
Since this is not an actual working piece..i think it make sense to comment it.

Reply:
I have fixed this in the latest commit. The code snippet is fully functional.